### DISCUSSION TITLE
2024-2025 - Practical work 3 - Alex Berberat, Lisa Gorgerat et Pierric Ripoll

### DISCUSSION THREAD

**Discussion by LisaGorgerat** [heig-vd-dai-course/heig-vd-dai-course#545] [Show and tell]
Projet en commun avec BDR:
Application de gestion de média avec des utilisateurs qui peuvent faire des listes personnalisées et intéragir avec les médias.
Seul la partie de gestion des médias serait développé en fonction de DAI.

**Comment by ludelafo** [comment]
Comme discuté en classe, tout bon pour nous.

**Comment by LisaGorgerat** [comment]
the link to our project: [https://github.com/Nyaaw/ProjectBDR](https://github.com/Nyaaw/ProjectBDR)
hash of our last commit: 0c6aa1c5d16d50c2677ed8807838e7a20951ca8f
release (because we need to keep working for bdr): rendu
ip de la vm: 4.180.67.252

**Comment by Nyaaw** [comment]
Bonsoir, je viens de désactiver la règle de protection du main de notre repo, celle qui oblige tous les commits à être signés. 
Ceci pour finir notre projet BDR. Merci de votre compréhension.
